functions/data_transformations.R

```diff
@@ -0,0 +1,24 @@
+transform_metadata_to_df <- function(data){
```

Comment:
Looks good! Add some more comments in the function to make it more clear what each line do.